src/services/qbit/QbitProvider.ts

```diff
@@ -563,8 +563,10 @@ export default class QBitProvider implements IProvider {
     return this.torrentAction('addTags', hashes, { tags: tags.join('|') })
   }
 
-  async removeTorrentTag(hashes: string[], tags: string[]): Promise<void> {
-    return this.torrentAction('removeTags', hashes, { tags: tags.join('|') })
+  async removeTorrentTag(hashes: string[], tags?: string[]): Promise<void> {
+    return tags
+      ? this.torrentAction('removeTags', hashes, { tags: tags.join(',') })
+      : this.torrentAction('removeTags', hashes)
```

Comment:
Reduce duplicated code
```suggestion
    let options = tags ? { tags: tags.join(',') } : undefined
    return this.torrentAction('removeTags', hashes, options)
```